ticdc/manage-ticdc.md

```diff
@@ -220,6 +220,24 @@ The following are descriptions of parameters that can be configured for the sink
 
 For more parameters of Pulsar, see [pulsar-client-go ClientOptions](https://godoc.org/github.com/apache/pulsar-client-go/pulsar#ClientOptions) and [pulsar-client-go ProducerOptions](https://godoc.org/github.com/apache/pulsar-client-go/pulsar#ProducerOptions).
 
+#### Configure sink URI with cdclog
+
+The `cdclog` files (files written by TiCDC) can be used together with Backup & Restore (BR) to provide point-in-time (PiTR) recovery. This is an experimental feature.
+
+See `br restore cdclog --help` for more information on this command.
+
+{{< copyable "shell-regular" >}}
+
+```shell
+cdc cli changefeed create --pd=http://10.0.10.25:2379 --sink-uri="local:///data/cdclog" --config changefeed.toml
+```
+
+{{< copyable "shell-regular" >}}
+
+```shell
+cdc cli changefeed create --pd=http://10.0.10.25:2379 --sink-uri="s3://logbucket/test?endpoint=http://$S3_ENDPOINT/" --config changefeed.toml
+```
```

Comment:
I think we should describe what the two commands are about. ("local:///data/cdclog" and "s3://logbucket/test?endpoint=http://$S3_ENDPOINT/")